### DISCUSSION TITLE
建议增加Mod收藏夹功能

### DISCUSSION THREAD

**Discussion by Cairl** [Meloong-Git/PCL#3169] [讨论]
这个游戏的Mod用量非常大，有的时候重装新版本要一个个Mod去搜很麻烦。能否添加一个收藏夹功能，用来给用户（或仅限赞助用户）收藏Mod的用处，这类信息的储存空间也不会有很大成本吧？

**Comment by MoYuan-CN** [comment]
#487 提及此事，请到 #1646 为其投票
